src/context/user/UserContext.jsx

```diff
@@ -6,6 +6,23 @@ const UserContext = createContext();
 export const UserProvider = ({children}) => {
 
   const [user,setUser] = useState({isLoggedIn: false});
+
+  /**
```

Comment:
commented code can be removed

Reply:
This was intentionally put to give a view how my user data will look after login.